README.md

```diff
@@ -1,5 +1,5 @@
 #PayPal IPN Listener
-
+[![Packagist](https://img.shields.io/packagist/v/mike182uk/paypal-ipn-listener.svg)](https://packagist.org/packages/mike182uk/paypal-ipn-listener)
```

Comment:
Could you add `?style=flat-square` on the end of `paypal-ipn-listener.svg` to keep consistency with the other `shields.io` badges.